client/src/main/java/io/pravega/client/admin/ReaderGroupManager.java

```diff
@@ -76,26 +77,29 @@ static ReaderGroupManager withScope(String scope, ClientConfig clientConfig) {
      * @return True if ReaderGroup was created.
      * @throws ConfigMismatchException If the reader group already exists with a different configuration. Use {@link ReaderGroup#resetReaderGroup} to change
      * the reader group configuration.
+     * @throws ObjectClosedException If the already closed ReaderGroupManager is used for further calls.
      */
-    boolean createReaderGroup(String groupName, ReaderGroupConfig config) throws ConfigMismatchException;
+    boolean createReaderGroup(String groupName, ReaderGroupConfig config) throws ConfigMismatchException, ObjectClosedException;
```

Comment:
Here and below, this is a runtime exception, so it doesn't need to be in the signature. Feel free to leave it in the Javadoc.

Reply:
Done.